.github/actions/javascript/verifySignedCommits/index.js

```diff
@@ -11736,6 +11736,7 @@ class GithubUtils {
                 isFirebaseChecked: issue.body ? /-\s\[x]\sI checked \[Firebase Crashlytics]/.test(issue.body) : false,
                 isGHStatusChecked: issue.body ? /-\s\[x]\sI checked \[GitHub Status]/.test(issue.body) : false,
                 version,
+                tag: `${version}-staging`,
```

Comment:
NAB I guess but after the 15th time you have to wonder if we can do this once somewhere 😄

Reply:
All the `actions/javascript/**/index.js` are compiled